.travis/osx..script.sh

```diff
@@ -1,8 +1,9 @@
 #!/usr/bin/env bash
 
-if [ $QT5 ]; then
+if [ "$QT5" ]; then
         # Workaround; No FindQt5.cmake module exists
-        export CMAKE_PREFIX_PATH="$(brew --prefix qt55)"
+        CMAKE_PREFIX_PATH="$(brew --prefix qt55)"
+        export CMAKE_PREFIX_PATH
 fi
 
-cmake $CMAKE_FLAGS -DUSE_WERROR=OFF ..
+cmake "$CMAKE_FLAGS" -DUSE_WERROR=OFF ..
```

Comment:
> CMake can correctly interpret this quoted thing, I have tested it

I have tested it and it does not, the same goes with `apt-get`. Putting quotes around will pass `$CMAKE_FLAGS` as a single argument. For example, `-DA=2 -DB=3` will be interpreted as if you passed `-DA="2 -DB=3"`.